scripts/world/World.gd

```diff
@@ -8,13 +8,14 @@ func _ready():
         level.connect("level_area_entered", self, "_on_level_area_entered")
         level.connect("level_area_exited", self, "_on_level_area_exited")
 
-    if Game.current_level:
-        $Player.position = get_node(Game.current_level.entrance_node).position
-        Game.current_level = null
+    if Game.current_level_id:
```

Comment:
Is this for when the player has just exited a level? Maybe we should move it to function that says as much, or add a comment.

Reply:
Yeah, it's so the player ends up on the same Level in World. Will extract it to function.